plugins/pulp_rpm/plugins/migrations/0027_remove_checksum_type_field.py

```diff
@@ -0,0 +1,21 @@
+"""
+This migration removes the `checksum_type` field of the units_rpm collection.
+"""
+from pulp.server.db import connection
+
+
+def migrate(*args, **kwargs):
+    """
+    Perform the migration as described in this module's docblock.
+
+    :param args:   unused
+    :type  args:   list
+    :param kwargs: unused
+    :type  kwargs: dict
+    """
+    db = connection.get_database()
+    collection = db['units_rpm']
+    collection.update(
+        {"checksum_type": {"$exists": True}},
```

Comment:
I don't see any documentation that says the query needs to be on an indexed field to update many documents at once. I would assume that I would see a warning in documentation or some questions about this on StackOverflow. However, I didn't find either.